docs/migrate-to-ory/auth0.mdx

```diff
@@ -0,0 +1,140 @@
+---
+id: auth0
+title: Guide to migrate from Auth0 to Ory
+sidebar_label: Auth0
+slug: migrating-from-auth0
+---
+
+## Migrating from Auth0 to Ory
+
+This guide helps you to migrate from Auth0 to Ory. The base example assumes that your Auth0 setup includes a database connection
+and email and password for credentials. For other setups this guide serves as a good starting point, which you can fit to your
+usecase.
+
+In this guide, you'll:
+
+1. open a support ticket to get your hashed passwords from Auth0.
+
+2. run a bulk user export to get your complete user data.
+
+3. create identities in Ory Network.
+
+### Export password hashes
+
+Because password hashes are considered sensitive information, Auth0 doesn't export them as part of the general export process. You
+must file a support ticket to get the hashes and other password related information.
+
+! **When you are exporting hashed passwords, neither you, nor Auth0 or the Ory system has access to the plain text password** !
```

Comment:
Please use Docusaurus admonitions for notes like this one: 

https://docusaurus.io/docs/markdown-features/admonitions

You can find many examples of these in our docs. ;)

Reply:
I won't add this comment to every applicable place, just make sure to do it for every note-thingy.